### PR TITLE
Rounding decimal places for csv output

### PR DESCRIPTION
**Pull request by NOAA-Strausz**
See if this meets your standards.

### CODE REVIEW

File: ARGOS_service_data_converter.py [hunk 1]
```diff
@@ -151,7 +154,7 @@ def __init__(self, missing=1e35):
       self.missing = missing
 
     @staticmethod
-    def get_data(fobj=None, time='current'):
```
Comment (shaunwbell) [inline ARGOS_service_data_converter.py]: This change will break the API and it is needed to get current data without specifying a date. Put it back in and resubmit.

**Comment by shaunwbell** [comment]
Not gonna worry about it this time but the “string interpolation” ‘%’ (modulo) operator is perhaps the least desired way to format strings. 

A more pythonic way and discussion can be found here https://www.python-course.eu/python3_formatted_output.php

format() function or f-strings (python 3.6+) should be used

File: ARGOS_service_data_converter.py (hunk 1 above)
Comment (NOAA-Strausz) [reply ARGOS_service_data_converter.py]: I didn't do this so something is weird.

Comment (shaunwbell) [reply ARGOS_service_data_converter.py]: its not really worth investigating on such a small program probably it was incidental or you copy pasted from a old file or something... doesn't matter, its small enough, i'll accept the changes and then fix it myself and push another commit.  It was useful enough to get a feel how to handle pull requests.

**Pull request merged**